DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/FilteredMgp/AssetComplete.cshtml

```diff
@@ -7,14 +7,14 @@
 }
 @section breadcrumbs {
   <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" asp-action="SelfAssessment" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])">Self Assessment</a></li>
-  <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" asp-action="FilteredRecommendations" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])">Recommended Learning Resources</a></li>
-  <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" asp-action="FilteredLearningAsset" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])" asp-route-assetId="@Model.Id">Learning Resource</a></li>
+  <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" asp-controller="RecommendedLearning" asp-action="RecommendedLearning" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])">Recommended Learning Resources</a></li>
+  <li class="nhsuk-breadcrumb__item"><a class="nhsuk-breadcrumb__link" asp-controller="RecommendedLearning" asp-action="FilteredLearningAsset" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])" asp-route-assetId="@Model.Id">Learning Resource</a></li>
 <li class="nhsuk-breadcrumb__item">Complete</li>
 }
 <link rel="stylesheet" href="@Url.Content("~/css/learningPortal/selfAssessment.css")" asp-append-version="true">
 
 <div class="nhsuk-back-link">
-  <a class="nhsuk-back-link__link trigger-loader" asp-action="FilteredLearningAsset" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])" asp-route-assetId="@Model.Id">
+  <a class="nhsuk-back-link__link trigger-loader" asp-controller="RecommendedLearning" asp-action="FilteredLearningAsset" asp-route-selfAssessmentId="@(ViewContext.RouteData.Values["selfAssessmentId"])" asp-route-assetId="@Model.Id">
```

Comment:
I've actively avoided running the formatter/breaking down the really long lines in these views in the FilteredMgp folder (except where it was really dumb) since these are the views for the old page that I'm replacing in 648 and I wasn't sure if it was really worth spending the time to do this. This applies to the button role too. I can go back and change them all if we want that though.

Reply:
Ah I see, now worries then!